bindings/wasm/identity_wasm/examples/src/0_basic/-1_test_api_call.ts

```diff
@@ -21,7 +21,7 @@ import { IotaClient } from "@iota/iota-sdk/client";
 import { getFaucetHost, requestIotaFromFaucetV0 } from "@iota/iota-sdk/faucet";
 import { Transaction } from "@iota/iota-sdk/transactions";
 import { IOTA_TYPE_ARG } from "@iota/iota-sdk/utils";
-import { IDENTITY_IOTA_PACKAGE_ID, NETWORK_NAME_FAUCET, NETWORK_URL, TEST_GAS_BUDGET } from "../utils_alpha";
+import { IDENTITY_IOTA_PACKAGE_ID, NETWORK_NAME_FAUCET, NETWORK_URL, TEST_GAS_BUDGET } from "../util";
```

Comment:
shall we delete this example here?

Reply:
Yes, removed it, as most parts covered by them should be covered by actual examples by now. ^^